### PR TITLE
fix: schema path checks

### PR DESCRIPTION
**Pull request by karstens**
* fix the getSchema function

### CODE REVIEW

**Comment by GarthDB** [comment]
## Run report for [c2a1f608](https://github.com/adobe/spectrum-tokens/commit/c2a1f608512d75400fb436b79157964a00e31e57)
Total time: 10s | Comparison time: 10.9s | Estimated savings: 853.7ms (7.9% faster)
|     | Action | Time | Status | Info |
| :-: | :----- | ---: | :----- | :--- |
| 🟩 | `SyncWorkspace` | 0ms | Passed |  |
| ⬛️ | `SetupNodeTool(~20.12)` | 1.2s | Skipped |  |
| 🟩 | `InstallNodeDeps(~20.12)` | 7.4s | Passed |  |
| 🟩 | `SyncNodeProject(componentSchemas)` | 0.2ms | Passed |  |
| 🟩 | `SyncNodeProject(root)` | 0.2ms | Passed |  |
| 🟥 | `RunTask(componentSchemas:test)` | 1.4s | Failed |  |
| 🟩 | `RunTask(root:test)` | 1.3s | Passed |  |

<details><summary><strong>Touched files</strong></summary><div>

```
packages/component-schemas/index.js
packages/component-schemas/test/index.test.js
```

</div></details>

**Comment by karstens** [comment]
@GarthDB I cannot reproduce the failing tests locally. It is fine on my machine. Do you know why this fails in the git action env? 

Local log:
```
➜  component-schemas git:(karstens/fix--schema-paths-check) ✗ pnpm moon run :test
▪▪▪▪ tokens:buildTokens (cached from previous run, 8e486338)
▪▪▪▪ componentSchemas:test
▪▪▪▪ root:test
componentSchemas:test |
            root:test |
componentSchemas:test |   ✔ index › getSlugFromDocumentationUrl should return last part of documentationUrl
componentSchemas:test |   ✔ index › getSlugFromDocumentationUrl should return last part of documentationUrl even without trailing slash
componentSchemas:test |   ✔ index › the number of schema returned by getAllSchemas should match the number of schema returned by schemaFileNames
componentSchemas:test |   ✔ index › getAllSlugs should return all component slugs
componentSchemas:test |   ✔ index › getSchemaBySlug should return a schema
componentSchemas:test |   ✔ index › getSchemaFile should fetch schema data
            root:test |   ✔ validate github workflow config files
            root:test |   ─
            root:test |
            root:test |   1 test passed
▪▪▪▪ root:test (1s 33ms, a601bc9d)
componentSchemas:test |   ✔ componentSchemaValidator › Every component schema should validate against the definition
componentSchemas:test |   ✔ componentSchemaValidator › component examples should validate against the definition
componentSchemas:test |   ✔ componentSchemaValidator › component schema should be valid
componentSchemas:test |   ─
componentSchemas:test |
componentSchemas:test |   9 tests passed
▪▪▪▪ componentSchemas:test (1s 117ms, 6a0eb89c)
▪▪▪▪ csvGenerator:test (cached from previous run, 1309f047)

  ✔ basic test
  ─

  1 test passed
▪▪▪▪ tokens:test (cached from previous run, b2011540)

  ✔ deprecateExpress › Deprecate Express should add deprecation metadata to Express tokens
  ✔ drover › Drover should match light variable values
  ✔ drover › Drover should match dark variable values
  ✔ drover › Drover should match darkest variable values
  ✔ checkManifest › check for uuids
  ✔ checkComponentProps › ensure all component tokens are have component data
  ✔ checkId › ensure uuids are unique
  ✔ checkPrivate › ensure all color-palette.json tokens are private
  ✔ schemaValidators › alias › Every token schema should validate against the definition
  ✔ tokenSchemaValidator › Every token should have a $schema property
  ✔ tokenSchemaValidator › Every token schema should validate against the definition
  ✔ tokenSchemaValidator › Schema should compile without errors
  ✔ checkUniqueTokenNames › check for duplicate token names across all token files (268ms)
  ─

  13 tests passed

Tasks: 5 completed (3 cached)
 Time: 1s 562ms
```

**Comment by GarthDB** [comment]
I'm merging this and I'll look at fixing it in the other branch.

File: packages/component-schemas/test/index.test.js [hunk 1]
```diff
@@ -28,7 +28,24 @@ test("the number of schema returned by getAllSchemas should match the number of
...
+  const relativeSchemaComponent = await getSchemaFile("component.json");
+  t.is(relativeSchemaComponent.title, "Component");
+
+  const relativeSchemaActionBar = await getSchemaFile(
+    "components/action-bar.json",
+  );
+  t.is(relativeSchemaActionBar.title, "Action bar");
+
+  const absoluteSchemaComponent = await getSchemaFile(
+    "/Users/karstens/Developer/github/adobe/spectrum-tokens/packages/component-schemas/schemas/component.json",
```
Comment (GarthDB) [inline packages/component-schemas/test/index.test.js:40]: This doesn't work in the GitHub actions container or on any computer other than @karstens'.

Comment (karstens) [reply packages/component-schemas/test/index.test.js:40]: @GarthDB Oh no, my stupid mistake. Sorry. Thanks for finding this.

**Pull request merged**